client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -35,34 +36,81 @@ const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 			'woocommerce-payments'
 		);
 
-	/* translators: <a> link to dispute documentation. %s is the clients claim for the dispute, eg "The cardholder claims this is an unrecognized charge." */
-	let noticeText = __(
-		'<strong>%s</strong> Challenge the dispute if you believe the claim is invalid, ' +
-			'or accept to forfeit the funds and pay the dispute fee. ' +
-			'Non-response will result in an automatic loss. <a>Learn more about responding to disputes</a>',
-		'woocommerce-payments'
-	);
-	let learnMoreDocsUrl =
-		'https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#responding';
+	// Format the deadline date
+	const dueByDate = moment
+		.unix( dispute.evidence_details?.due_by ?? 0 )
+		.format( 'MMM D, YYYY h:mm A' );
+
+	// Get the bank name from payment method details
+	const getBankName = () => {
```

Comment:
I'd add some Jest tests for the bank name display logic, but just realized that we don't have tests for any of the dispute-details components